Use pkg-config instead of imlib2-config because the current version of imlib2
doesn't have imlib2-config anymore.
--- configure.ac.orig	2004-09-04 09:42:10.000000000 -0500
+++ configure.ac	2023-01-21 04:27:09.000000000 -0600
@@ -16,16 +16,10 @@
 AM_MAINTAINER_MODE
 AM_WITH_DMALLOC
 
-AC_PATH_GENERIC(imlib2, 1.0.0, [
+PKG_CHECK_MODULES([IMLIB], [imlib2 >= 1.0.0], [
   AC_SUBST(IMLIB_LIBS)
   AC_SUBST(IMLIB_CFLAGS) ],
-  AC_MSG_ERROR(Cannot find imlib2: Is imlib2-config in the path?) )
-
-dnl the above doesn't work for some reason :/
-IMLIB_LIBS=`imlib2-config --libs`
-IMLIB_CFLAGS=`imlib2-config --cflags`
-AC_SUBST(IMLIB_LIBS)
-AC_SUBST(IMLIB_CFLAGS)
+  AC_MSG_ERROR(Cannot find imlib2) )
 
 LIBS="$LIBS -lm"
 CFLAGS="$CFLAGS -Wall"
